src/main/entities.ts

```diff
@@ -0,0 +1,14 @@
+import LoggerService from "./services/logger.service";
+import AppService from "./services/app.service";
+
+
+/**
+ * List of classes what you want to be autoInjectable
```

Comment:
Sorry for being OCD, would you mind adding a period at the end of each sentence?